sys/net/gnrc/network_layer/ipv6/nib/_nib-internal.c

```diff
@@ -83,8 +83,9 @@ void _nib_release(void)
 static inline bool _addr_equals(const ipv6_addr_t *addr,
                                 const _nib_onl_entry_t *node)
 {
-    return (addr == NULL) || ipv6_addr_is_unspecified(&node->ipv6) ||
-           (ipv6_addr_equal(addr, &node->ipv6));
+    return (addr == NULL) ?
+    ipv6_addr_is_unspecified(&node->ipv6)
+    : (ipv6_addr_equal(addr, &node->ipv6));
```

Comment:
```suggestion
    if (addr == NULL) {
        return ipv6_addr_is_unspecified(&node->ipv6);
    } else {
        return ipv6_addr_equal(addr, &node->ipv6);
    }
```

makes this much more readable